keyboards/centromere/keymaps/mini/keymap.c

```diff
@@ -0,0 +1,161 @@
+#include QMK_KEYBOARD_H
+
+
+// Each layer gets a name for readability, which is then used in the keymap matrix below.
+// The underscores don't mean anything - you can have a layer called STUFF or any other name.
+// Layer names don't all need to be of the same length, obviously, and you can also skip them
+// entirely and just use numbers.
+#define _QWERTY 0
+#define _LOWER 1
+#define _RAISE 2
+#define _ADJUST 16
```

Comment:
```suggestion
enum centromere_layers
{
    _QWERTY,
    _LOWER,
    _RAISE,
    _ADJUST
};
```

I would suggest using this `enum` construct instead, for a few reasons:

1. It's easier to insert a new layer without having to re-index everything
2. Having the last layer as 16 with artificially inflate the firmware size by creating "phantom" layers 3 through 15.
3. Indexing is handled automatically by the `enum`.

Reply:
@ridingqwerty Done.